docs/sources/visualizations/node-graph.md

```diff
@@ -111,6 +111,6 @@ Optional fields:
 
 | Field name    | Type          | Description                                                                                                                                                                                                                 |
 | ------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| mainStat      | string/number | First stat shown in the overlay when hovering over the edge. Can be either string in which case the value will be shown as it is or it can be a number in which case any unit associated with that field will be also shown |
-| secondaryStat | string/number | Same as mainStat but shown right under it.                                                                                                                                                                                  |
+| mainstat      | string/number | First stat shown in the overlay when hovering over the edge. Can be either string in which case the value will be shown as it is or it can be a number in which case any unit associated with that field will be also shown |
```

Comment:
```suggestion
| mainstat      | string/number | First stat shown in the overlay when hovering over the edge. It can be a string showing the value as is or it can be a number. If it is a number,  any unit associated with that field is also shown |
```